scripts/release/workflow/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
```

Comment:
We should be using the more broad `set -euo pipefail` in all bash scripts to avoid surprises. See [explanation](https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425) (`-x` is a little too much IMO and is not related to errors).

Reply:
Added to this and all bash scripts with `set -o errexit`